modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/SegmentsExperimentsDetails.es.js

```diff
@@ -68,6 +69,28 @@ function SegmentsExperimentsDetails({segmentsExperiment}) {
 						<dd className="d-inline inline-item-after text-secondary">
 							{indexToPercentageString(confidenceLevel)}
 						</dd>
+
+						<div className="c-my-2">
+							<dt className="d-inline-block">
+								{Liferay.Language.get('test-type') + ':' + ' '}
+							</dt>
+
+							<dd className="d-inline inline-item-after text-secondary">
+								{Liferay.Language.get('standard')}
+							</dd>
```

Comment:
@interaminense if I use sub here all the entire text is going to be bold because it is wrapped inside a <dt></dt> element:

`<dt className="d-inline-block">
							{sub(Liferay.Language.get('test-type-x'), [
								Liferay.Language.get('dynamic'),
							])}
						</dt>`
						
And the mockup asks to set `dynamic` part as font-normal....is it really essential to use `sub` here?

Reply:
Ok, but can you use a literal string instead?

Reply:
done